parsons/zoom/zoom.py

```diff
@@ -34,7 +34,7 @@ def refresh_header_token(self):
         # on JWT generation using Zoom API: https://marketplace.zoom.us/docs/guides/auth/jwt
 
         payload = {"iss": self.api_key, "exp": int(datetime.datetime.now().timestamp() + 30)}
-        token = jwt.encode(payload, self.api_secret, algorithm='HS256').decode("utf-8")
+        token = jwt.encode(payload, self.api_secret, algorithm='HS256')
```

Comment:
Did you still intend to take the `decode` away, or was that a leftover from debugging?

Reply:
That's intentional! I think the original error came from the fact that, sometime over the summer, the PyJWT package upgraded its `encode()` method to return a string instead of bytes. Then the whole package ambiguity obscured that very simple fix.